src/main/java/com/sleekbyte/tailor/Tailor.java

```diff
@@ -141,9 +143,12 @@ public static void printSummary(long numFiles, long numSkipped, long numErrors,
 
                 if (className.equals(FileListener.class.getName())) {
                     continue;
-                } else if (className.equals(CommentAnalyzer.class.getName())) {
```

Comment:
maybe it's time to use a switch statement here .. I'll leave this up to your discretion